modules/stats.php

```diff
@@ -36,6 +36,16 @@
 	'tag_base'
 );
 
+function is_dnt_enabled() {
+	foreach ($_SERVER as $name => $value) {
```

Comment:
Could you add some spaces here, as explained [here](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)?